test/jdk/java/awt/color/ICC_ColorSpace/SimpleSRGBToFromCIEXYZ.java

```diff
@@ -39,20 +40,11 @@ public static void main(String[] args) {
             float[] inv = cs.fromCIEXYZ(xyz);
 
             if (inv[0] != 0 || Math.abs(inv[1] - g) > 0.0001f || inv[2] != 0) {
-                System.err.println("Expected color:\t" + toString(rgb));
-                System.err.println("XYZ color:\t\t" + toString(xyz));
-                System.err.println("Actual color:\t" + toString(inv));
-                throw new Error("Wrong color");
+                System.err.println("Expected color:\t" + Arrays.toString(rgb));
+                System.err.println("XYZ color:\t\t" + Arrays.toString(xyz));
+                System.err.println("Actual color:\t" + Arrays.toString(inv));
+                throw new java.lang.Error("Wrong color");
```

Comment:
Is the full class name required? All classes from `java.lang` are imported, aren't they?

Reply:
for test execution it does not matter but when opened in ide it conflicts with some other Error classes from the tests.

Reply:
> for test execution it does not matter but when opened in ide it conflicts with some other Error classes from the tests.

I feel your pain!

I suggest refactoring those tests and make their private `Error` class a static member of the test class itself.

This is why I'm for having only one public class in a test file and no other top-level classes — to avoid such naming clashes, which makes it easier to edit and run tests from an IDE.